pycloudlib.toml.template

```diff
@@ -24,10 +24,15 @@ tenant_id = ""
 # key_name = ""  # Defaults to your username if not set
 
 [ec2]
-# Most values can be found in ~/.aws/credentials or ~/.aws/config
-access_key_id = ""  # in ~/.aws/credentials
-secret_access_key = ""  # in ~/.aws/credentials
 region = ""  # in ~/.aws/config
+# If 'aws configure sso' has been run, 'profile' should be the only credentials needed
+profile = ""  # in ~/.aws/config
+
+# With modern SSO, these should no longer be necessary.
```

Comment:
Nit: `With modern SSO` sounds a bit pretentious and likely prone to age bad. Something like the following?

```suggestion
# If profile is given, these are not necessary.
```